### PR TITLE
Updated README.md file  and five other files

### PR DESCRIPTION
**Pull request by aoamusat**
Updated the README.md files on the program dependencies like (NLTK, vincent)
updated some .py files to compatible with python 3 print() function

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,3 +2,7 @@
 Some programs doing the basic functions in Tweepy (Twitter for Python).
 
 A sample python.json file has been included.
+
+# Required Dependency
```
Comment (Parth-Vader) [inline README.md:6]: A `requirements.txt` file with `pip install -r requirements.txt` would also be great in addition to this, do you agree?

Comment (aoamusat) [reply README.md:6]: Yeah. I agree with you. I was thinking about that earlier

File: requirements.txt [hunk 2]
```diff
@@ -0,0 +1,2 @@
+nltk 
+vincent
```
Comment (Parth-Vader) [inline requirements.txt]: `tweepy` is missing.

**Comment by Parth-Vader** [comment]
@dev-haykay Thanks a lot mate. Added a minor comment.

**Comment by aoamusat** [comment]
@Parth-Vader Can you please merge this PR

File: requirements.txt (hunk 2 above)
Comment (Parth-Vader) [reply requirements.txt]: @dev-haykay This needs to be added.

**Comment by aoamusat** [comment]
You mean tweepy?

**Comment by Parth-Vader** [comment]
Yes. It needs to be added in `requirements.txt`.

**Comment by aoamusat** [comment]
tweepy is now added.

**Comment by Parth-Vader** [comment]
@dev-haykay Thanks a lot. Merging. :+1:

**Pull request merged**